doc/sources/gettingstarted/installation.rst

```diff
@@ -190,6 +192,14 @@ The script will download and build the SDL dependencies from source. It will als
 the dependencies into a directory named `kivy-dependencies`. This directory will be used
 by Kivy to build and install Kivy from source with SDL support.
 
+On macOS is still possible to build SDL2 with the Apple-provided OpenGL framework, but it is
```

Comment:
Minor typo /is still possible/it is still possible/

Reply:
Suggestion applied! Thank you!